packages/vue/src/components/range/RangeSlider.jsx

```diff
@@ -0,0 +1,200 @@
+import { Actions, helper } from "@appbaseio/reactivecore";
+import Container from "../../styles/Container";
+import { connect } from "../../utils/index";
+import Title from "../../styles/Title";
+import types from "../../utils/vueTypes";
+import VueTypes from "vue-types";
+import vueSlider from 'vue-slider-component'
+
+const {
+  addComponent,
+  removeComponent,
+  watchComponent,
+  updateQuery,
+  setQueryListener,
+  setQueryOptions
+} = Actions;
+
+const { isEqual, checkValueChange, getClassName, pushToAndClause } = helper;
+
+const RangeSlider = { 
+ 
+  name: "RangeSlider",
+  components: {
+    vueSlider
+  },
+  data(){
+    this.state = {
+    currentValue: [this.$props.range.start, this.$props.range.end],
+    stats:[]
+  }
+  this.locked = false;
+    return this.state;
+  },
+
+  props:{
+    beforeValueChange: types.func,
+    className: VueTypes.string.def(""),
+    range: VueTypes.shape({
+      start: VueTypes.integer.def(0),
+      end: VueTypes.integer.def(10),
+    }),
+    componentId: types.stringRequired,
+    customQuery: types.func,
+    data: types.data,
+    dataField: types.stringRequired,
+    defaultSelected: types.string,
+    filterLabel: types.string,
+    innerClass: types.style,
+    react: types.react,
+    showFilter: VueTypes.bool.def(true),
+    showCheckbox: VueTypes.bool.def(true),
+    title: types.title,
+    URLParams: VueTypes.bool.def(false)
+  },
+
+  methods:{
+
+    setReact(props) {
+      if (props.react) {
+        this.watchComponent(props.componentId, props.react);
+      }
```

Comment:
In the else part `this.watchComponent(props.componentId, {})` ?

Reply:
sure, but any specific reason for doing so?

Reply:
I needed it in dynamicslider, to watch for componentId if react prop was not defined.
@bietkul Can give more details on this